releasenotes/notes/add-QM.remove_interaction-e97a7ac91db2e2b8.yaml

```diff
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    Implement ``QuadraticModel.remove_interaction`` method, previously it
+    raised a ``NotImplementedError`` error.
```

Comment:
If you're splicing with a comma, then "method, which previously raised"

Reply:
Too late. It's in the git history like that forever. Haunting you.

Reply:
Well, you're stuck forever with https://github.com/dwavesystems/dimod/pull/947#issuecomment-896239578